pkg/serveswagger/swagger_test.go

```diff
@@ -0,0 +1,27 @@
+package serveswagger
+
+import (
+	"os/exec"
+	"regexp"
+	"testing"
+)
+
+func TestCheckTimestamps(t *testing.T) {
+	cmd := exec.Command("git diff ../../pkg/serverswagger")
```

Comment:
needs separate args, and `serverswagger` is a typo

Reply:
IIRC I had trouble with the apple version of `git` and ended up installing a different version - probably with brew.

Reply:
Before running `git diff`, this needs to re-generate the code with `go generate ./cmd/go-swagger/restapi`. Otherwise, it's not actually testing anything.